e2e/wdio/helper/assertion-helper.ts

```diff
@@ -37,5 +37,30 @@ export function checkBorderColor(array, expectedColor): void {
     });
 }
 
+export function checkElementDisplayed(element: string): any {
+    const elLength = webDriver.getElementArrayLength(element);
+    for (let i = 0; elLength > i; i++) {
+        expect(webDriver.elementDisplayed(element, i)).toBe(true);
+    }
+}
 
+export function checkElementText(element: string): any {
+    const elLength = webDriver.getElementArrayLength(element);
+    for (let i = 0; elLength > i; i++) {
+        expect(webDriver.getText(element, i)).not.toBe(null, '');
```

Comment:
It's better to check the actual text than checking that text is not null

Reply:
for action-list-item I added a check for specific text 
for standard-list-item and object-list-item I want to keep the current check because there are really a lot of examples with a lot of unique text and all that will be covered by screenshot tests when we add it. For now, I'm just checking that some kind of data is there to make sure the element isn't empty (it's like title, price, status, weight, additional data, additional status, etc).